sections/futures/LeverageInput/LeverageInput.tsx

```diff
@@ -50,6 +50,7 @@ const LeverageInput: FC<LeverageInputProps> = ({ onLeverageChange }) => {
 		return position?.remainingMargin.lte(0) || maxLeverage.lte(0);
 	}, [position, maxLeverage]);
 
+	const leverageButtons = maxLeverage.eq(25) ? ['5', '10', '25'] : ['2', '5', '10'];
```

Comment:
I think the maxLeverage value changes when positions are open, right? So handling those cases might need us to do things that are trickier than this.

Reply:
The value of `maxLeverage` is handled here https://github.com/Kwenta/kwenta/blob/dev/store/futures/index.ts#L108